prime-router/src/main/kotlin/azure/SendFunction.kt

```diff
@@ -33,16 +33,16 @@ class SendFunction(private val workflowEngine: WorkflowEngine = WorkflowEngine()
         message: String,
         context: ExecutionContext,
     ) {
+        val actionHistory = ActionHistory(TaskAction.send, context)
+        actionHistory.trackActionParams(message)
         try {
```

Comment:
This move of the actionHistory code is part of overall changes to SendFunction so that the actionHistory gets stored in the database, even on major Exceptions.